scss/_patterns_code-snippet.scss

```diff
@@ -22,7 +22,11 @@ $color-snippet-heading-bg: rgba($color-x-dark, 0.08);
         content: '';
         left: $sph-inner;
         position: absolute;
-        top: 0.85rem;
+        top: 0.8rem;
+
+        @media (min-width: $breakpoint-x-large) {
```

Comment:
Technically this should be wrapped in `@if ($increase-font-size-on-larger-screens) {`, as increase of the font size can be configured.

Reply:
And put a comment to explain why we need this exception

Reply:
TIL!

Reply:
> TIL!

yep, me too ;)